crates/cosmos/cosmos-relayer/src/contexts/build.rs

```diff
@@ -184,8 +189,10 @@ impl CosmosBuilder {
 
         let event_source_mode = chain_config.event_source.clone();
 
-        let tx_config = TxConfig::try_from(&chain_config)?;
+        let mut tx_config = TxConfig::try_from(&chain_config)?;
 
+        // TODO: Eventually use Config to define dynamic gas config
+        tx_config.gas_config.dynamic_gas_config = self.dynamic_gas.clone();
```

Comment:
Actually, we can modify `build_chain_with_config` to accept an additional parameter `dynamic_gas_config`.

The chain is being built [here](https://github.com/informalsystems/hermes-sdk/blob/main/crates/cosmos/cosmos-integration-tests/src/impls/bootstrap/build_cosmos_chain.rs#L48), and you can use `Bootstrap: HasDynamicGasConfig` to get the config.

Reply:
Done in d534f6125ef2e2418625457fa9d06422aa4de014